wiki/Tournaments/RDOC/2014/en.md

```diff
@@ -0,0 +1,305 @@
+
+## Organisation
+
+The Royal Danish osu!Cup 2014 was run by various community members.
+
+| Position | Member(s) |
+| :-- | :-- |
+| Manager | ![][flag_DK] [Mayoi Hachikuji](https://osu.ppy.sh/users/2114273) |
+| Map selector | ![][flag_DK] [Mayoi Hachikuji](https://osu.ppy.sh/users/2114273), ![][flag_DK] [Heibel](https://osu.ppy.sh/users/983000) |
+| Referee | ![][flag_DK] [Mayoi Hachikuji](https://osu.ppy.sh/users/2114273), ![][flag_DK] [Heibel](https://osu.ppy.sh/users/983000) |
+| Other helpers | ![][flag_DK] [Vocalover](https://osu.ppy.sh/users/2281840), ![][flag_SE] [Kotayo](https://osu.ppy.sh/users/1730025), ![][flag_DK] [PlasticSmoothie](https://osu.ppy.sh/users/296565) |
+| Designer | ![][flag_DK] [Nickyeh](https://osu.ppy.sh/users/2314944) |
+| Donator | ![][flag_DK] [Gupply](https://osu.ppy.sh/users/2983311), ![][flag_DK] [TimG](https://osu.ppy.sh/users/1879963) |
+
+## Links
+
+- [Discussion thread](https://osu.ppy.sh/community/forums/topics/179395)
+- [Challonge brackets](https://challonge.com/OSUDK)
+
+## Participants
+
+| Members |
+| :-- |
+| ![][flag_DK] [Gupply](https://osu.ppy.sh/users/2983311), ![][flag_DK] [TimG](https://osu.ppy.sh/users/1879963), ![][flag_DK] [Jobbe](https://osu.ppy.sh/users/2707230), ![][flag_DK] raww, ![][flag_DK] [Otaku-kun123](https://osu.ppy.sh/users/3664366), ![][flag_DK] [Tenasow](https://osu.ppy.sh/users/2735417), ![][flag_DK] [Lethly](https://osu.ppy.sh/users/1637208), ![][flag_DK] [Nicklz3107](https://osu.ppy.sh/users/1902855), ![][flag_DK] [winthermens](https://osu.ppy.sh/users/2080544), ![][flag_DK] [Nickyeh](https://osu.ppy.sh/users/2314944), ![][flag_DK] [Kylling](https://osu.ppy.sh/users/2691925), ![][flag_DK] [Amou Kanade](https://osu.ppy.sh/users/3111445), ![][flag_DK] [-Dustbunny-](https://osu.ppy.sh/users/3711672), ![][flag_DK] [Shoujo](https://osu.ppy.sh/users/35608), ![][flag_DK] [Nithirot](https://osu.ppy.sh/users/3053982), ![][flag_DK] [jeross](https://osu.ppy.sh/users/2744290), ![][flag_DK] [arild159](https://osu.ppy.sh/users/3089925), ![][flag_DK] [ZoDiaN1337](https://osu.ppy.sh/users/2256249), ![][flag_DK] [powerupgm](https://osu.ppy.sh/users/2954652), ![][flag_DK] [Deaxz](https://osu.ppy.sh/users/2742880), ![][flag_DK] [karrusel](https://osu.ppy.sh/users/2429098), ![][flag_DK] [Broniel](https://osu.ppy.sh/users/3175330), ![][flag_DK] [Tropians](https://osu.ppy.sh/users/2536611), ![][flag_DK] [Maltahl](https://osu.ppy.sh/users/2369766), ![][flag_DK] Shiryuu, ![][flag_DK] [Tonarinototoro](https://osu.ppy.sh/users/2678812), ![][flag_DK] [Mr Proncoz](https://osu.ppy.sh/users/2491276), ![][flag_DK] [nitram166](https://osu.ppy.sh/users/2552515), ![][flag_DK] [Ximadi](https://osu.ppy.sh/users/2700938), ![][flag_DK] [PlasticSmoothie](https://osu.ppy.sh/users/296565), ![][flag_DK] [Kazutakee](https://osu.ppy.sh/users/2637514), ![][flag_DK] [Kozhler](https://osu.ppy.sh/users/2245929) |
+
+## Podium
+
+ This competition has come to an end and resulted in the following podium:
```

Comment:
```suggestion
This competition has come to an end and resulted in the following podium:
```
It seems you have more blanks than i can imagine. Maybe recheck every iteration for this line specifically.